db/migrate/20230401123456_add_new_columns_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddNewColumnsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :vip, :boolean, default: false
+    add_column :users, :role, :string
+    add_column :users, :team_member_id, :bigint
+    add_index :users, :team_member_id
+  end
```

Comment:
Adding a new column with an index is good for performance, but ensure that the 'team_member_id' column is also added as a foreign key constraint to maintain referential integrity.